### DISCUSSION TITLE
Transferring domain from GoDaddy to Vercel

### DISCUSSION THREAD

**Discussion by ben-sommer** [vercel/community#509] [Help] [answered]
When I try to transfer my domain from GoDaddy to Vercel, it asks for the `IPS tag on new registrar` but I can't find any reference to Vercel's IPS tag in the docs - how can I transfer the domain without knowing this?
![image](https://user-images.githubusercontent.com/32485726/159135852-ba93f562-1ca5-422c-be13-a23d9bc2d38c.png)

**Comment by swarnava** [comment, marked as answer]
Hi, is this a co.uk domain?

**Reply by ben-sommer** [reply]
Yes, it is. Just saw [this reply](https://github.com/vercel/vercel/discussions/5591#discussioncomment-234009) which states that you don't support `.co.uk` inbound domain transfers currently - is this in your roadmap?

Thanks

**Reply by swarnava** [reply]
Thanks for confirming. Unfortunately, we don't have any concrete timeline for this.
